charts/kube-state-metrics/templates/servicemonitor.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
   {{- end }}
 spec:
   jobLabel: {{ default "app.kubernetes.io/name" .Values.prometheus.monitor.jobLabel }}
+{{ include "servicemonitor.scrapeLimits" .Values.prometheus.monitor | indent 2 }}
```

Comment:
indent looks wrong

Reply:
huh? it's on the same level like `jobLabel`?

Reply:
Indent is fine after templating but you could align it with `jobLabel` without any change on the output.
However, this generate a blank line before the block, not sure if this could be an issue, any idea @monotek ?

```
spec:
  jobLabel: foo
  
  sampleLimit: 1
  targetLimit: 2
  [...]
```

Reply:
> Indent is fine after templating but you could align it with `jobLabel` without any change on the output. However, this generate a blank line before the block, not sure if this could be an issue, any idea @monotek ?
> 
> ```
> spec:
>   jobLabel: foo
>   
>   sampleLimit: 1
>   targetLimit: 2
>   [...]
> ```

@dotdc 
It's because the code use `{{ ` which does not auto delete the whitespace (`{{- ` does, see my suggestion below).